aws/region/regions.go

```diff
@@ -8,26 +8,75 @@ var nameToCode = map[string]Code{
 	"US West (Oregon)":           "us-west-2",
 	"US West (Los Angeles)":      "us-west-2-lax-1", // Included in the offer file but missing from the docs.
 	"Canada (Central)":           "ca-central-1",
+	"Canada West (Calgary)":      "ca-west-1",
 	"EU (Stockholm)":             "eu-north-1",
 	"EU (Ireland)":               "eu-west-1",
 	"EU (London)":                "eu-west-2",
 	"EU (Paris)":                 "eu-west-3",
 	"EU (Frankfurt)":             "eu-central-1",
+	"EU (Zurich)":                "eu-central-2",
 	"EU (Milan)":                 "eu-south-1",
-	"Asia Pacific (Mumbai)":      "ap-south-1",
+	"EU (Spain)":                 "eu-south-2",
 	"Asia Pacific (Tokyo)":       "ap-northeast-1",
 	"Asia Pacific (Seoul)":       "ap-northeast-2",
 	"Asia Pacific (Osaka-Local)": "ap-northeast-3",
 	"Asia Pacific (Singapore)":   "ap-southeast-1",
 	"Asia Pacific (Sydney)":      "ap-southeast-2",
+	"Asie Pacific (Jakarta)":     "ap-southeast-3",
+	"Asia Pacific (Melbourne)":   "ap-southeast-4",
+	"Asie Pacific (Malaysia)":    "ap-southeast-5",
+	"Asie Pacific (Thailand)":    "ap-southeast-7",
 	"Asia Pacific (Hong Kong)":   "ap-east-1",
+	"Asia Pacific (Mumbai)":      "ap-south-1",
+	"Asie Pacific (Hyderabad)":   "ap-south-2",
 	"South America (Sao Paulo)":  "sa-east-1",
 	"China (Beijing)":            "cn-north-1",
 	"China (Ningxia)":            "cn-northwest-1",
 	"AWS GovCloud (US-West)":     "us-gov-west-1",
 	"AWS GovCloud (US-East)":     "us-gov-east-1",
 	"Middle East (Bahrain)":      "me-south-1",
 	"Africa (Cape Town)":         "af-south-1",
+	"Israel (Tel Aviv)":          "il-central-1",
+	"Middle East (UAE)":          "me-central-1",
+}
+
+var codeToShortName = map[string]string{
```

Comment:
Where is this list from? Just in case we need to update it.

Reply:
I took it from the S3 pricing JSON file, but it also appears to be available here: [AWS Usage Report Documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/aws-usage-report-understand.html).

It seems to be a common pattern across services, but I noticed an exception for us-east-1, which sometimes appears as "US" or remains empty.

I'll set "US" as the default and handle exceptions as needed, then update the documentation accordingly.